### ISSUE TITLE
Win95 crash

### ISSUE THREAD

**Issue by siordon** [schlae/snark-barker-mca#9]
Has the Snark Barker MCA been tested on Windows 95? 
I have a model 56 (configured with joystick port disabled), and Win95 causes the whole system to lock if the Sound Blaster device is enabled. There are no conflicts, and everything checks out ok with SBDIAG.  Most games I've tested (DOS) work fine, except for Epic Pinball which also causes the system to lock up.
Any suggestions?

**Comment by siordon** [comment]
More data points - may be port related. 
Syndicate
- Works with no port specified, with sound.  (Eg: main /iirq5 /idma1)
- Works with wrong port specified, no sound. (Eg: main /iirq5 /idma1 /iio240)
- Crashes with correct port specified. (Eg: main /iirq5 /idma1 /iio220)
- If card changed to port 240, I get no sound even with the first example

Epic Pinball
- Works with incorrect port specified, reverts to PC speaker
- Crashes with correct port specified (tried with both 220 and 240)

Win95
- Yellow exclamation mark if configured with incorrect port, but at least it doesn't crash.

**Comment by vossi1** [comment]
Hi,
1. Epic Pinball runs perfectly in my 9595 with P90 Complex. In the setup I selected SB Mono and use parameters from SoundBlaster Variable.
2. I tried to install Windows 95 (before I had the Snark Barker) from harddisk like I read in the web. But I had no success and give up. The cdrom does not work because it's a special scsi cdrom from IBM - that's why it's only possible from HD.

**Comment by siordon** [comment]
Thanks. 
1. Epic Pinball setup is correct (SB Mono, matches setup/blaster environment variable).  When I run it, there is a brief "click" from the speakers (the usual sound often made when a game initialises a soundcard), the internal speaker beeps, and the computer freezes completely (power button required)
2. Mine has no CDROM. I installed Win95 by copying the files to the hard drive from an external parallel port CDROM drive. Win95 installed just fine. However with the SB installed, enabled, and configured correctly, the system locks up completely when trying to start Windows (or, if the SB device is disabled, when enabling the device)

**Comment by holzachr** [comment]
Hi,
I'm having the same issue with Win95(B) on my 9576i with Lacuna planar and 486DX2/66. Computer locks up when enabling the device, or while booting.
SBDIAG is fine, and sound comes out of DOS DOOM with IRQ 2, but crashes with any other IRQ.
Tried all possible IO/IRQ/DMA settings, but Win95 keeps locking up.
That's strange, as I am 100 % sure I had a similar machine (9577i Lacuna, so it's the same planar) running fine with the SB **Pro** MCV a few years ago.
Unfortunately, there's no updated drivers out there for Win95, apart from the builtin ones MS delivered with Windows, so nothing one could try on that front.

**Comment by holzachr** [comment]
BTW, it's a card built by Monotech, and it's running fine under WfW 3.11.

**Comment by vossi1** [comment]
Hi, I just installed win95c on my 9595 P66. Win 95 hangs after MSSBLST.VXD loading.
Is there really no other win95 driver?
Maybe I will try win98...

**Comment by holzachr** [comment]
I heard, the trick is installing the Windows 3.x driver under 95, but didn't verify it myself yet.

**Comment by vossi1** [comment]
Yes, the Win3.1 driver works for all software that uses the windows driver.
You have to install the oemsetup.inf from the sbmcv/win31 folder vie hardware/manual/disk.
Only install the second driver it offers (wave & midi). This is the digital audio and midi-out driver. It does not appear in the controlpanel/system, but you find it in controlpanel/multimedia (sb 1.5 wave). You can also install directX8 if you have a pentium. dxdiag shows the driver also. 
Do NOT install the first driver (sb midi synth). It freezes the machine instantly if at access. So you have sadly no midi music via the OPL chip. I tried the adlib opl-driver - but it also doesnt work.
Some games have their own driver (ex. 3d ultrapinball) - they have sadly no sound with the snark barker.
PS: With a 486 you can only use Direct x 6.1.
PS2: I installed the microsoft joystick driver from the hardware list - seems to work. But I have no working 558 in this card - I hope the next batch arrives and is finally ok.
